doc/users/next_whats_new/new_color_cycle.rst

```diff
@@ -0,0 +1,4 @@
+New colorblind-friendly color cycle
+===================================
+
+A new color cycle has been added, tableau-colorblind10, to provide another option for colorblind-friendly plots.
```

Comment:
Suggest "A new style sheet has been adding defining a color cycle ``tableu_colorblind10`` ..."  

Suggest a link to: :ref:`sphx_glr_tutorials_introductory_customizing.py`  (Umm, I think I referenced that correctly)